deploy/docker/production/supervisord.conf

```
@@ -0,0 +1,54 @@
+[supervisord]
+pidfile=%(here)s/supervisord.pid
+logfile_maxbytes=50MB
+logfile_backups=10
+loglevel=info
+nodaemon=true
+minfds=1024
+minprocs=200
+user=nginx
+
+[program:cgap1]
+autorestart=true
+startsecs=6
+command=pserve production.ini http_port=6543
+stdout_logfile=/dev/stdout
+stdout_logfile_maxbytes=0
+stderr_logfile_maxbytes=0
+redirect_stderr=true
+
+[program:cgap2]
+autorestart=true
+startsecs=6
+command=pserve production.ini http_port=6544
+stdout_logfile=/dev/stdout
+stdout_logfile_maxbytes=0
+stderr_logfile_maxbytes=0
+redirect_stderr=true
+
+[program:cgap3]
+autorestart=true
+startsecs=6
+command=pserve production.ini http_port=6545
+stdout_logfile=/dev/stdout
+stdout_logfile_maxbytes=0
+stderr_logfile_maxbytes=0
+redirect_stderr=true
+
+[program:cgap4]
+autorestart=true
+startsecs=6
+command=pserve production.ini http_port=6546
+stdout_logfile=/dev/stdout
+stdout_logfile_maxbytes=0
+stderr_logfile_maxbytes=0
+redirect_stderr=true
+
+[program:cgap5]
+autorestart=true
+startsecs=6
+command=pserve production.ini http_port=6547
+stdout_logfile=/dev/stdout
+stdout_logfile_maxbytes=0
+stderr_logfile_maxbytes=0
+redirect_stderr=true
```

Needs a final newline.
```suggestion
redirect_stderr=true

```